cli/index.js

```diff
@@ -81,4 +431,425 @@ y.command({
     }
 })
 
+// create login command
+y.scriptName("connectme")
+y.command({
+    command: 'login',
+    describe: 'login with an account',
+    builder: {
+        username: {
+            describe: 'Username',
+            demandOption: true,
+            type: 'string'
+        },
+        password: {
+            describe: 'Password of the account'
+        }
+        // TODO: prompt to confirm the password
+    },
+    handler(argv) {
+        login(argv)
+    }
+})
+
+// create guest account command
+y.scriptName("connectme")
+y.command({
+    command: 'guest',
+    describe: 'Creates a guest account ',
+    builder: {
+        username: {
+            describe: 'Username',
+            demandOption: true,
+            type: 'string'
+        },
+        userID: {
+            describe: 'An user ID',
+            demandOption: true,
+            type: 'string'
+        },
+       picture: {
+           describe: 'A profile picture',
+            demandOption: true,
+            type: 'string'
+        }
+        // TODO: prompt to confirm the password
+    },
+    handler(argv) {
+        guest(argv)
+    }
+})
+
+// Logout user
+y.scriptName("connectme")
+y.command({
+    command: 'logout',
+    describe: 'Logout current account ',
+    builder: {
+        username: {
+            describe: 'Username',
+            demandOption: true,
+            type: 'string'
+        },
+        userID: {
+            describe: 'An user ID',
+            demandOption: true,
+            type: 'string'
+        },

```

Comment:
again, you don't need to force the user to specify the userId, it should be enough to remember the username.

Reply:
done